### ISSUE TITLE
Font calculation gives an error: "is not a valid CSS value"

### ISSUE THREAD

**Issue by soundstep** [sass/node-sass#735]
**node sass version**
2.0.1

**ruby sass version:**
3.4.13

**OSX**
10.9.1

**node**
v0.10.35

-------------------------------

I found this issue updating the grunt-sass npm module to its latest version (v0.18.0) which has node-sass 2.0.1 as a dependency.

The following code is executing correctly with the ruby sass library but I got an error with the node-sass library.

```
./node_modules/node-sass/bin/node-sass main.scss main.css
```

```
{
  "status": 1,
  "file": "main.scss",
  "line": 25,
  "column": 14,
  "message": "6.25px/px/rem is not a valid CSS value",
  "code": 1
}
```

Here is the code used:

https://gist.github.com/soundstep/abad27d4ac44682b0d05

Test the code:

http://codepen.io/soundstep/pen/EapbOx

**Comment by soundstep** [comment]
Another useful information, the code describe above was also working correctly with the node-sass 0.8.6

**Comment by am11** [comment]
@soundstep, thank you for reporting this issue. As mentioned in the [bug reporting guidelines](https://github.com/sass/node-sass/blob/master/CONTRIBUTING.md#bug-reports), the syntax related issues should be reported to libsass. You have everything ready, just clone this issue to https://github.com/sass/libsass. :)

**Comment by soundstep** [comment]
@am11 No worries. Could you point me out how I can find which libsass version you are using in the node-sass 2.0.1?

Cheers.

**Comment by am11** [comment]
From code usage:

```javascript
require('node-sass').info()
```

From CLI:

```bash
node_modules/.bin/node-sass --help # at the top of output
```

**Comment by am11** [comment]
Also note that starting from v3 (the upcoming version of node-sass), it will be bit different:

Code:

```javascript
console.log(require('node-sass').info)
```

CLI:

```bash
node_modules/.bin/node-sass --version 
```

**Comment by soundstep** [comment]
Thanks, much appreciated!
I'll report the issue there.

Cheers.

**Comment by soundstep** [comment]
The issue is apparently fixed in libsass 3.2:
https://github.com/sass/libsass/issues/917

Any idea when the 3.2 will be implemented? Any pull request on the way with the 3.2?

**Comment by am11** [comment]
I am also wait for libsass v3.2.0 to land here: https://github.com/sass/libsass/releases, so it would be right after that! :)
